### DISCUSSION TITLE
Relation between d and c in definitions of "condition described by" and "describes condition"

### DISCUSSION THREAD

**Discussion by nklsbckmnn** [CommonCoreOntology/CommonCoreOntologies#904] [Ontology Terms and Class Schema]
"condition described by" is defined as "c condition_described_by p iff p is an instance of a Performance Specification, and p has part d, and d is an instance of a Descriptive Information Content Entity, and p prescribes an entity s, and c is an entity that is causally relevant to s existing as prescribed by p."

"describes condition" is defined as "p describes_condition c iff p is an instance of a Performance Specification, and p has part d, and d is an instance of a Descriptive Information Content Entity, and p prescribes an entity s, and c is an entity that is causally relevant to s existing as prescribed by p."

Isn't the relation between d and c "describes"? If so, shouldn't this be stated in the definitions?

**Comment by mark-jensen** [comment]
@eliasweatherfield Thanks for catching that! Will update the definitions and push up changes soon.

**Comment by mark-jensen** [comment]
closed with dd18eb09e9dcf0258b0780cdc8a284491e78dc79

**Comment by alanruttenberg** [comment]
Performance Specification: 
> A Directive Information Content Entity that prescribes some aspect of the behavior of a participant in a Process given one or more operating conditions.

The domain of 'describes condition' is  Performance Specification. However the definition seems to say something much more general:

> p describes_condition c iff p is an instance of a Performance Specification and c is an instance of Entity and p has part d, and d is an instance of a Descriptive Information Content Entity, and p prescribes an entity s, and d describes c, and c is an **entity** that is causally relevant **to s existing** as prescribed by p. [my emphasis]

Given the documentation for *Performance Specification* should s be a process and c a participant in that process?

This came up as I tried to answer https://github.com/w3c/sdw-sosa-ssn/pull/401#issuecomment-3815072409

**Comment by johnbeve** [comment]
@alanruttenberg For clarity, I've paired down the definition to: 

> p describes condition c iff p is a Performance Specification that has part Descriptive Information Content Entity d that describes Entity c and c is causally relevant to the existence of Entity s as prescribed by p. 

I can read this four different ways; all of which seem within plausible interpretations of the Performance Specification class definition, given how broad it is. My understanding of performance specifications, generally, are specifications of required outcomes for a project or process to be considered acceptable. 

Suppose I want a strawberry milkshake and require it have real strawberries. Here are two readings: 
- milkshake specification p describes condition strawberries c iff p has a part that describes strawberries c that are causally relevant to the existence of my (acceptable) milkshake s that is prescribed by p.
- milkshake specification p describes condition strawberries c iff p has a part that describes strawberries c that are causally relevant to the existence of a(n acceptable) milkshake preparation process s that is prescribed by p. 

On the first reading, s is a continuant and c likely a proper part of that continuant. 
On the second reading, s is a process and c a participant in that process. 

I think both readings are in tension with the definition of Performance Specification, since what is prescribed is behavior of a participant rather than the participant itself. 

Two more readings: It doesn't seem quite right to say p describes condition strawberries as opposed to p describes condition 'having strawberries', which suggests a process. On this reading, we have: 
- milkshake specification p describes condition having strawberries c iff p has a part that describes having strawberries c that is causally relevant to the existence of my (acceptable) milkshake s that is prescribed by p.
- milkshake specification p describes condition having strawberries c iff p has a part that describes having strawberries c that is causally relevant to the existence of a(n acceptable) milkshake preparation process s that is prescribed by p. 

On this third reading, s is a continuant but c is likely a process part of the history of s.
On this fourth reading, s is a process and c is likely a process part of the history of s.

Here the tension with the definition of Performance Specification is gone, but so has direct reference to the participant (strawberries) in the process. In any event, here c is on the occurrent side. 

Caveat: I am not advocating to keep the existing definition. I think it's tortured and should be replaced. I prefer my refinement but it is also fourfold ambiguous, so would need to be revised. 

@mark-jensen @neilotte @APCox
